src/entity/enemy_basic.js

```diff
@@ -1,29 +1,38 @@
 import { Entity } from "./entity.js";
 import { enemyBlueprint } from "./blueprints.js";
+import { G_LOGGER } from "../logger.js";
 
 class EnemyBasic extends Entity  {
     constructor() {
         super(enemyBlueprint);
-        this.elapsed = 0;
+
+        // set maxvelocity to 10
+        this.preStates["movement"]["maxVelocity"] = 10.0;
     }
 
-    attack(delta, curPos, player, enemies, space) {
+    attack(delta, curPosX, curPosY, player, enemies, space) {
```

Comment:
That is a good idea but for these functions we should not. The reasoning is because these functions are the mutator functions and should use no libraries and no ES6, only strict browser Javascript code to keep our interpreter as light as possible. Might be useful for future iterations though!

Reply:
Changed so that any object in class entity class will have properly bounded variables. So PIXI can now be accessed via `this.helpers.Pixi` as defined in `Entity` base-class constructor.